[2024-04-23T11:20:40.691+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-08T00:00:00+00:00 [queued]>
[2024-04-23T11:20:40.697+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-08T00:00:00+00:00 [queued]>
[2024-04-23T11:20:40.697+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-23T11:20:40.709+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): gen_dates> on 2024-03-08 00:00:00+00:00
[2024-04-23T11:20:40.713+0000] {standard_task_runner.py:60} INFO - Started process 75332 to run task
[2024-04-23T11:20:40.716+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_eq', 'gen_dates', 'scheduled__2024-03-08T00:00:00+00:00', '--job-id', '245', '--raw', '--subdir', 'DAGS_FOLDER/kbs/custom_dags/pipeline_eq.py', '--cfg-path', '/tmp/tmpx1_mdgz8']
[2024-04-23T11:20:40.716+0000] {standard_task_runner.py:88} INFO - Job 245: Subtask gen_dates
[2024-04-23T11:20:40.740+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-08T00:00:00+00:00 [running]> on host datalab-yasmine
[2024-04-23T11:20:40.768+0000] {logging_mixin.py:188} WARNING - /home/yasmine/.local/share/virtualenvs/knowledge-base-siaap-WvBZnYs-/lib/python3.8/site-packages/airflow/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-04-23T11:20:40.876+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_eq' AIRFLOW_CTX_TASK_ID='gen_dates' AIRFLOW_CTX_EXECUTION_DATE='2024-03-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-08T00:00:00+00:00'
[2024-04-23T11:20:40.876+0000] {rw.py:84} INFO - Starting to save files into Minio
[2024-04-23T11:20:42.698+0000] {python.py:202} INFO - Done. Returned value was: dates_20240308
[2024-04-23T11:20:42.711+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=pipeline_eq, task_id=gen_dates, execution_date=20240308T000000, start_date=20240423T112040, end_date=20240423T112042
[2024-04-23T11:20:42.736+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T11:20:42.765+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-23T11:24:12.215+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-08T00:00:00+00:00 [queued]>
[2024-04-23T11:24:12.220+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-08T00:00:00+00:00 [queued]>
[2024-04-23T11:24:12.220+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-23T11:24:12.232+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): gen_dates> on 2024-03-08 00:00:00+00:00
[2024-04-23T11:24:12.236+0000] {standard_task_runner.py:60} INFO - Started process 80606 to run task
[2024-04-23T11:24:12.239+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_eq', 'gen_dates', 'scheduled__2024-03-08T00:00:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/kbs/custom_dags/pipeline_eq.py', '--cfg-path', '/tmp/tmp8ae8_uzj']
[2024-04-23T11:24:12.239+0000] {standard_task_runner.py:88} INFO - Job 267: Subtask gen_dates
[2024-04-23T11:24:12.263+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-08T00:00:00+00:00 [running]> on host datalab-yasmine
[2024-04-23T11:24:12.291+0000] {logging_mixin.py:188} WARNING - /home/yasmine/.local/share/virtualenvs/knowledge-base-siaap-WvBZnYs-/lib/python3.8/site-packages/airflow/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-04-23T11:24:12.416+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_eq' AIRFLOW_CTX_TASK_ID='gen_dates' AIRFLOW_CTX_EXECUTION_DATE='2024-03-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-08T00:00:00+00:00'
[2024-04-23T11:24:12.417+0000] {logging_mixin.py:188} INFO - {'start': '2024-03-08 00:00:00+00:00', 'end': '2024-04-23 11:24:12'}
[2024-04-23T11:24:12.417+0000] {rw.py:84} INFO - Starting to save files into Minio
[2024-04-23T11:24:13.974+0000] {python.py:202} INFO - Done. Returned value was: dates_20240308
[2024-04-23T11:24:13.988+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=pipeline_eq, task_id=gen_dates, execution_date=20240308T000000, start_date=20240423T112412, end_date=20240423T112413
[2024-04-23T11:24:14.018+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T11:24:14.032+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
